packages/node_modules/@ciscospark/react-component-utils/src/metrics.js

```diff
@@ -36,6 +36,7 @@ export function sendPerfMetric({key, data}, spark) {
       hostname: window.location.hostname,
       pathname: window.location.pathname,
       widgetVersion: process.env.REACT_CISCOSPARK_VERSION,
+      jsSdkVersion: spark.version,
```

Comment:
Yea, we should probably do that there, but in efforts to get this out into the wild before the weekend, can you create a ticket to do that in the SDK in the next sprint?